pkgs/by-name/li/libbladeRF/package.nix

```diff
@@ -62,7 +62,7 @@ stdenv.mkDerivation rec {
   ];
 
   env = lib.optionalAttrs stdenv.cc.isClang {
-    NIX_CFLAGS_COMPILE = "-Wno-error=unused-but-set-variable";
+    NIX_CFLAGS_COMPILE = "-Wno-error=unused-but-set-variable -Wno-error=tautological-overlap-compare";
```

Comment:
Same here. I never had direct access to Darwin machines along with the motivation to do so to test this thoroughly, but it'd be great to debug these build failures with upstream. They seem to be pretty active. Also, cc @cyclic-pentane per https://github.com/Nuand/bladeRF/pull/984#issuecomment-3382860239 - I assume they have some relevant knowledge :).